docs/01-overview/README.md

```diff
@@ -8,6 +8,12 @@ Kyma is an application runtime that provides you a flexible and easy way to conn
 
 ![overview](./assets/kyma-overview.svg)
 
+## Modular Kyma
+
+Classic Kyma offers a fixed set of mandatory components whose development rhythm is synchronized and determined by the release schedule. With the modular approach, Kyma components become modules, each providing one functionality developed independently of the other ones. With each successive release, fewer components will be available within the preconfigured Kyma, but more and more will be offered as independent modules.
```

Comment:
I thought it was possible to only install certain components when you use the CLI?